src/kit.ts

```diff
@@ -796,14 +793,13 @@ async function scanDirForClangForMSVCKits(dir: string, vsInstalls: VSInstallatio
         }
 
         const clangKits: Kit[] = [];
-        vsInstalls.forEach(vs => {
+        for (const vs of vsInstalls) {
```

Comment:
this may help with the problem I had in PR https://github.com/microsoft/vscode-cmake-tools/pull/2627. I will check it after merging this branch.